components/server/src/routes/auth.py

```diff
@@ -106,8 +106,14 @@ def user(username: str, email: str) -> str:
 @bottle.post("/api/v3/login")
 def login(database: Database) -> Dict[str, Union[bool, str]]:
     """Log the user in. Add credentials as JSON payload, e.g. {username: 'user', password: 'pass'}."""
-    username, password = get_credentials()
-    verified, email = verify_user(username, password)
+    forward_auth_enabled = bool(os.environ.get("FORWARD_AUTH_ENABLED", None))
```

Comment:
The None is not needed here because it's the default value of the "default" parameter of get().